action.yaml

```diff
@@ -130,51 +128,42 @@ runs:
         sed -i "s/sonar_scan=x/sonar_scan=${icon}/g" $RUNNER_TEMP/icons.properties
 
     # Citadel section
-    - name: Prepare citadel Json file
-      if: ${{ env.TRIGGER_CITADEL_SCAN }}
-      shell: bash
-      run: |
-        pip3 install pyyaml==6.0.1
-
-        python3 ${GITHUB_ACTION_PATH}/citadel/citadel.py \
-          "${GITHUB_WORKSPACE}/citadel-config.yaml" "${{ env.PROJECT_PATH }}" "$GITHUB_WORKFLOW_REF" "$RUNNER_TEMP"
-        
-        echo "CITADEL_JSON_PATH=$RUNNER_TEMP/citadel.json" >> $GITHUB_ENV
-
     - name: Upload to the citadel host
       if: ${{ env.CITADEL_JSON_PATH }}
       id: citadel 
       shell: bash
-      run: |        
-        file_paths="${{ env.CITADEL_JSON_PATH }}"
-        # unset CITADEL_JSON_PATH so that it doesn't get trigger again by other steps
-        echo "CITADEL_JSON_PATH=" >> $GITHUB_ENV
-        IFS=', ' read -ra file_paths_array <<< "$file_paths"
-        for file_path in "${file_paths_array[@]}"; do 
-          echo "Processing File $file_path"
-          cat $file_path
-          curl -vvvX POST '${{ env.CITADEL_URL }}' --silent --fail --show-error -d @$file_path -H 'Content-Type: application/json' -H "Connection: close"  
+      run: |
```

Comment:
Keep it how it was. You probably need to rebase

Reply:
updated